### PR TITLE
maybe fix CI

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

File: .github/workflows/build-and-publish-rule-book.yaml [hunk 1]
```diff
@@ -123,7 +123,7 @@ jobs:
           extra_system_packages: "inkscape font-liberation font-hack"
           latexmk_use_lualatex: ${{ matrix.luaengine }}
           latexmk_shell_escape: true
-          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && pdflatex --shell-escape main_${{ matrix.language }}.tex
+          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && lualatex --shell-escape main_${{ matrix.language }}.tex
```
Comment (qwrtln) [inline .github/workflows/build-and-publish-rule-book.yaml:126]: Are you sure about this? Only one file builds with lualatex

Comment (DarthGandalf) [reply .github/workflows/build-and-publish-rule-book.yaml]: oops

File: .github/workflows/build-and-publish-rule-book.yaml [hunk 2]
```diff
@@ -123,7 +123,7 @@ jobs:
           extra_system_packages: "inkscape font-liberation font-hack"
           latexmk_use_lualatex: ${{ matrix.luaengine }}
           latexmk_shell_escape: true
-          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && lualatex --shell-escape main_${{ matrix.language }}.tex
+          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && ${{ matrix.luaengine && 'lualatex' || 'pdflatex' }} --shell-escape main_${{ matrix.language }}.tex
```
Comment (qwrtln) [inline .github/workflows/build-and-publish-rule-book.yaml:126]: No, this won't do. GitHub doesn't support boolean type, both of those are true.
```suggestion
          post_compile: echo "icu_locale \"${{ matrix.language }}\"" >> index_style.ist && upmendex -s index_style main_${{ matrix.language }} && sed -i 's@\(\\noindent\\textbf{\)\(.\)@\\noindent\\textbf{\2}@g' main_${{ matrix.language }}.ind && ${{ matrix.luaengine == 'true' && 'lualatex' || 'pdflatex' }} --shell-escape main_${{ matrix.language }}.tex
```

Comment (DarthGandalf) [reply .github/workflows/build-and-publish-rule-book.yaml:122]: https://docs.github.com/en/actions/learn-github-actions/expressions#literals and my test commit disagree with you

**Review by qwrtln — APPROVED** [review]

**Pull request merged**